### PR TITLE
Add more alias support

### PR DESCRIPTION
**Pull request by nkzawa**

### CODE REVIEW

File: src/util/domains/setup-domain.ts [hunk 1]
```diff
@@ -8,14 +8,16 @@ import Client from '../client';
 import maybeGetDomainByName from './maybe-get-domain-by-name';
 import purchaseDomainIfAvailable from './purchase-domain-if-available';
 import verifyDomain from './verify-domain';
+import extactDomain from '../alias/extract-domain';
```
Comment (TooTallNate) [inline src/util/domains/setup-domain.ts]:
```suggestion
import extractDomain from '../alias/extract-domain';
```

File: src/util/domains/setup-domain.ts [hunk 2]
```diff
@@ -8,14 +8,16 @@ import Client from '../client';
...
 
 export default async function setupDomain(
   output: Output,
   client: Client,
   alias: string,
   contextName: string
 ) {
-  const parsedDomain = psl.parse(alias);
+  const aliasDomain = extactDomain(alias);
```
Comment (TooTallNate) [inline src/util/domains/setup-domain.ts]:
```suggestion
  const aliasDomain = extractDomain(alias);
```

**Review by TooTallNate — CHANGES_REQUESTED** [review]

**Review by TooTallNate — CHANGES_REQUESTED** [review]
The code looks good to me, but can we add an integration test for this?

**Review by AndyBitz — APPROVED** [review]

File: src/util/certs/get-wildcard-cns-for-alias.ts [hunk 3]
```diff
@@ -1,15 +1,21 @@
 import psl from 'psl';
 import { InvalidDomain } from '../errors-ts';
+import isWildcardAlias from '../alias/is-wildcard-alias';
+import extractDomain from '../alias/extract-domain';
 
-export default function getWildcardCNSForDomain(rawDomain: string) {
-  const parsedDomain = psl.parse(rawDomain);
+export default function getWildcardCNSForAlias(alias: string) {
```
Comment (TooTallNate) [inline src/util/certs/get-wildcard-cns-for-alias.ts:6]: I like the camel casing like in the file above better.

```suggestion
export default function getWildcardCnsForAlias(alias: string) {
```

**Review by TooTallNate — APPROVED** [review]

**Comment by leo** [comment]
Please remember to add a PR description @nkzawa.

**Pull request merged**